.eslintrc.json

```diff
@@ -12,6 +17,9 @@
       "error",
       "windows"
     ],
+    "no-console": [
+      "off"
+    ],
     "quotes": [
       "error",
       "double"
```

Comment:
Any particular reason for the preference of double quotes? I usually prefer single quotes myself and is what I typically see being used.

Reply:
.eslintrc.json was already in your project! I didn't change the `"quotes"` preference, I assumed this file had the settings you wanted.

I prefer single quotes too. I can change that.

Reply:
Look at that I did lol. I think when I originally built this extension I looked for a basic setup, and it must have included that, just never used it yet :)